cubedash/_stac.py

```diff
@@ -1012,6 +1012,71 @@ def collections():
     )
 
 
+@bp.route("/queryables")
+def queryables():
+    """
+    Define what terms are available for use when writing filter expressions for the entire catalog
+    Part of basic CQL2 conformance for stac-api filter implementation.
+    See: https://github.com/stac-api-extensions/filter#queryables
+    """
+    return _utils.as_json(
+        {
+            "$schema": "http://json-schema.org/draft-07/schema#",
+            "$id": "https://example.org/queryables",
+            "type": "object",
+            "title": "",
+            "properties": {
+                "id": {
+                    "title": "Item ID",
+                    "description": "Item identifier",
+                    "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/"
+                    "item.json#/definitions/core/allOf/2/properties/id",
+                }
```

Comment:
Actually/also, according to the specification document, this should include all of the STAC Item fields. 

> It is required to implement both of these endpoints, but for a STAC API, this may simply be a static document of the STAC-specific fields. A basic Queryables definitions for STAC Items **should include at least the fields id, collection, geometry, and datetime**.